openage/convert/value_object/read/media/smx.pyx

```diff
@@ -59,6 +57,415 @@ cdef public dict LAYER_TYPES = {
     1: SMXLayerType.SHADOW,
     2: SMXLayerType.OUTLINE,
 }
+cdef class SMXMainLayer8to5(SMXLayer):
```

Comment:
You should declare all classes that inherit from `SMXLayer` **after** `SMXLayer` has been declared as you would do normally in Python/C++ code. Otherwise, this may create compilation problems in the future (even if it doesn't do now).